integration/src/integration/java/pl/allegro/tech/hermes/integration/PublishingAvroTest.java

```diff
@@ -361,6 +361,32 @@ public void shouldUseExplicitSchemaVersionWhenPublishingAndConsuming() {
         });
     }
 
+    @Test
+    public void shouldUseExplicitSchemaVersionWhenPublishingAndConsumingWithLowercaseHeader() {
+        // given
+        Topic topic = topic("explicitSchemaVersion.topic")
+                .withContentType(AVRO)
+                .withSchemaVersionAwareSerialization()
+                .build();
+        operations.buildTopicWithSchema(topicWithSchema(topic, load("/schema/user.avsc").toString()));
+        operations.createSubscription(topic, "subscription", HTTP_ENDPOINT_URL, ContentType.AVRO);
+
+        operations.saveSchema(topic, load("/schema/user_v2.avsc").toString());
+
+        // when
+        HermesMessage message = hermesMessage(topic.getQualifiedName(), user.asBytes())
+                .withContentType(AVRO_BINARY)
+                .withHeader("schema-version", "1")
```

Comment:
will the test pass if you remove this header? hermes frontend uses latest schema version if no header is available.

Reply:
Test fails if you remove it with error "Invalid message: Could not deserialize avro message with provided schema"

Reply:
The same error like previously.

Reply:
OK thx :)